packages/py3_pip.rb

```diff
@@ -23,6 +23,7 @@ class Py3_pip < Package
      x86_64: '729a9958cae998936dc3ca300feb7c10eb8d56f74b524913cc84cb1cf6bbe553'
   })
 
+  depends_on 'python3'
   depends_on 'py3_setuptools'
```

Comment:
Should this line include `=> :build`?

Reply:
Pip requires setuptools to install packages.

Reply:
So it's runtime dependency here and nowhere else?  That doesn't make any sense.

Reply:
Yes. Pip requires setuptools to install packages and won't work without it.